v2/pkg/protocols/dns/request.go

```diff
@@ -47,6 +48,12 @@ func (request *Request) ExecuteWithResults(input string, metadata /*TODO review
 	variablesMap := request.options.Variables.Evaluate(vars)
 	vars = generators.MergeMaps(variablesMap, vars)
 
+	if request.options.Options.Debug || request.options.Options.DebugResponse {
+		gologger.Debug().Msgf("Protocol request variables")
+		vardump.Variables(vars)
```

Comment:
It would be better to control a single log message from one place. E.g.`vardump.Variables` (the renamed version of it) would return a string, instead of hiding the logging logic inside.

Reply:
Having one single log call for logically belonging messages is also important, otherwise another thread might write something on the output, causing inconsistency. We should also add a colon at the end of the "variables" to signal that the message continues on the next line.